src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs

```diff
@@ -49,6 +49,7 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
                 {
                     this.Height = minHeightSize;
                 }
+                if (_vm.Children != null && _vm.Children.Count == 0) this.Width = defaultWidthSize;
```

Comment:
Comments please, and our team usually do not push if statements on same line

Reply:
Done.